### DISCUSSION TITLE
Merged Major Updates, Standardization, and Next Steps

### DISCUSSION THREAD

**Discussion by MickLesk** [community-scripts/ProxmoxVE#841] [Announcements]
# Dear Contributors and Collaborators  and others,

We are pleased to announce that **PR #710** and **PR #643** have been successfully merged! 🎉  

### 🚀 Features  
- 🏷️ **CT.sh Tagging**: Tags can now be assigned to CT.sh files and are transferred to Proxmox.  
- 📜 **Extended MOTD**: Enhanced display information.  
- ⚙️ **Base Default Settings**: Predefined settings in CT.sh files (customizable if needed).  
- 🌐 **IPv4/IPv6 Ping Extension**: Improved network testing functionality.  
- 🎨 **Optimized UI/UX**: Updated color schemes and improved navigation (Default/Advanced settings).  
- 🔄 **Spinner Update**: Smoother and more efficient loading animation.  
- 🐧 **Ubuntu 24.10 Support**: Added compatibility.  
- 🏔️ **Improved Alpine Support**: Transition to stable 3.20 and enhanced installation scripts.  
- ⚙️ **Default Settings with verbose**: Now its possible to activate the verbose directly in whiptail dialogue. 

### 🔧 Changes  
- 🗂️ **Streamlined CT.sh**: Old versions remain compatible.  
- 🛠️ **Resource Queries**: Prompt for updates even with system constraints.  
- 💬 **Whiptail Dialogs**: Adapted for a better user experience.  
- 🌟 **Future Preparations**: Enhanced readiness for Fedora, CentOS, RockLinux, AlmaLinux, and openSUSE as LXC installations.  

### 🐛 Bugfixes  
- ❌ **Error Handling**: Improved robustness.  
- 🪛 **Variable Declaration**: Clean and standardized.  
- 🖨️ **Optimized Echo Outputs**: Simplified and clarified script output messages.  

---

## 🧾 Standardization Efforts  

Aligned with **PR #643**, the following standardizations have been implemented:  
- 📄 **Unified Headers**: Consistent across all scripts.  
- ✅ **Completion Standardization**: Uniform “Completed” indications.  
- 🏷️ **TAGs in CT Scripts**: Every CT now supports tagging.  
- 🔗 **License & Author Merging**: License and author details now consolidated into a single line.  
- ⚙️ **Default Values**: Standardized and differentiated based on privileges.  
- 🌍 **Origin Source URL**: Every script now includes its source.  
- 📏 **Indentation Alignment**: Consistent formatting across scripts.  

---

## 📅 Next Steps  

🚀 In **Part 3**, we’ll introduce a Contributor Guide and Templates. A first draft will be shared soon, and we encourage your feedback and participation!  

🙏 Thank you for your contributions and patience as we enhance our standards and processes. Let’s continue building amazing solutions together!  

Best regards,  
*The Community-Scripts Team*  

---

Some impressions
![image](https://github.com/user-attachments/assets/7760965c-4c79-4bd6-9ec5-7a6425b1cb92)

![image](https://github.com/user-attachments/assets/5a03f925-21e8-40e4-8c83-48d0ea114197)

![image](https://github.com/user-attachments/assets/3579117c-2df9-4992-a2e4-759a0cd3238a)

**Comment by Popcorncandy09** [comment]
When i change the IP of the CT, the motd doesn't reflect this.
